backend/benefit/users/api/v1/views.py

```diff
@@ -31,3 +40,33 @@ def _get_current_user(self, request):
         if settings.NEXT_PUBLIC_MOCK_FLAG and isinstance(request.user, AnonymousUser):
             set_mock_user_name(request.user)
         return request.user
+
+
+class UserUuidGDPRAPIView(GDPRAPIView):
+    """
+    GDPR-API view that is used from Helsinki-profiili to query what data this app has
+    on person.
+    """
+
+    permission_classes = [BFGDPRScopesPermission]
+    authentication_classes = []
+
+    def get_object(self) -> AbstractBaseUser:
+        """Get user by Helsinki-profiili UUID that is stored as username."""
```

Comment:
"Helsinki-profiili UUID" is very slightly inaccurate description as this is the user ID received from Helsinki profiili and that user ID is an UUID, see https://github.com/City-of-Helsinki/yjdh/pull/1569/files#r1094424211

Maybe "Helsinki-profiili user ID" or "Helsinki profile user ID" would be more accurate. If you wanted you could also document that there's a "Helsinki profile profile ID" also which is a different UUID value than the "Helsinki profile user ID" just to make it exactly clear to the user what's going on here, but that's up to you.